AppDB/appscale/datastore/backup/datastore_backup.py

```diff
@@ -190,31 +190,6 @@ def get_entity_batch(self, first_key, batch_size, start_inclusive):
 
     return batch
 
-  def verify_entity(self, key, txn_id):
```

Comment:
I don't think I understand why this is no longer needed. Do you mind explaining that?

Reply:
It was left over from before we started writing entity data on commit. We used to write entity data before a transaction was committed and keep track of its status with the combination of a blacklist and a list of running transactions in ZooKeeper. Since 3.0, we no longer use this function.

Reply:
Got it! Thanks for clarifying that!